x-pack/filebeat/input/httpjson/value_tpl.go

```diff
@@ -62,13 +62,15 @@ func (t *valueTpl) Unpack(in string) error {
 			"mul":                 mul,
 			"div":                 div,
 			"hmac":                hmacStringHex,
+			"hash":                hashStringHex,
 			"base64Encode":        base64Encode,
 			"base64EncodeNoPad":   base64EncodeNoPad,
 			"base64Decode":        base64Decode,
 			"base64DecodeNoPad":   base64DecodeNoPad,
 			"join":                join,
 			"sprintf":             fmt.Sprintf,
```

Comment:
I noticed this exists; it is already provided by the text/template packages global template functions via `printf`.